public/app/plugins/panel/geomap/editor/LayersEditor.tsx

```diff
@@ -74,12 +69,12 @@ export const LayersEditor = (props: LayersEditorProps) => {
 
       <LayerDragDropList
         layers={layers}
+        showActions={() => layers.length > 2} // 2 because base layer is not counted!
```

Comment:
Having threshold set to `> 2` means that user is not able to delete a layer if there is only one layer. Is that expected / desired? The `shouldRenderDragIconLengthThreshold` LayerDragDropList should handle determining whether or not to show re-ordering icon

Is there anytime we don't want to display actions on geomap layers (as we don't have nested frame concept here?) If I'm understanding correctly we should just return `true` here

<img width="617" alt="Screen Shot 2022-05-17 at 8 08 38 PM" src="https://user-images.githubusercontent.com/22381771/168949388-f0cf3c65-ab46-4fc4-98ab-3c220917b609.png">

Reply:
Yes -- this is intentional.  The panel requires at least one layer to make any sense.  Since the base layer counts as a layer... we get >2

Reply:
is `shouldRenderDragIconLengthThreshold` ever anything becues >1?